app/views/users/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="background">
+
+
+      <div class="container">
+        <div>
+          <h1 class="title text-center display-6 text-muted">BUDGE<small style="color: #24a0ed;">TIFY</small></h1>
+        </div>
+        
+        <div class="row gap-3 padding centered">
+          <%= link_to 'Log in', user_session_path, class: 'btn btn-info text-nowrap row'%>
+          <%= link_to 'Sign up', new_user_registration_path, class: 'btn btn-light text-nowrap row'%>
+        </div>
+      
+    </div>
+   <span></span>
+   <span></span>
```

Comment:
- [ ] I would recommend keeping the color same as given for the splash screen. It will help in your real work to stick with the design what client or company require.

### Your design
![Screenshot from 2022-04-27 15-33-29](https://user-images.githubusercontent.com/26767383/165499965-4eafd7bf-66b5-4791-ad8d-8b5888d20d03.png)


### Given design
![Screenshot from 2022-04-27 15-33-21](https://user-images.githubusercontent.com/26767383/165500008-a4936e84-2735-47f4-8338-59b9fbcea649.png)